response_operations_ui/templates/errors/500-error.html

```diff
@@ -0,0 +1,15 @@
+{% extends "layouts/base.html" %}
```

Comment:
Did this error page come from RAD?
Definitely a lot better than what it was before :+1:

Reply:
Copied this from ras-frontstage, this'll probably be restyled down the line